java/src/main/resources/local/application-aws.yml

```diff
@@ -0,0 +1,10 @@
+aws:
+  xray:
+    enabled: ${AWS_XRAY_ENABLED:false}
+  secretsmanager:
+    enabled: ${AWS_SECRETS_ENABLED:false}
+
+# AWS Secrets Manager imports
+spring.config.import:
+  - optional:aws-secretsmanager:/stacks-secret/example-1/
+  - optional:aws-secretsmanager:/stacks-secret/example-2/
```

Comment:
Possibly requires a newline, check it

`python3 -m yamllint -sc yamllint.conf . yamllint.conf`